src/zcx_aff_tools.clas.abap

```diff
@@ -0,0 +1,49 @@
+CLASS zcx_aff_tools DEFINITION
```

Comment:
Shall we add `_root` to the exception class to indicate that this is the root exception?

Reply:
as of now, there is only one exception class, so not worth it IMHO

Reply:
Then let's stay with `zcx_aff_tools`.

Reply:
Fine with me